sdk/metric/aggregator/array/array.go

```diff
@@ -68,12 +70,11 @@ func (c *Aggregator) Quantile(q float64) (core.Number, error) {
 	return c.checkpoint.Quantile(q)
 }
 
-func (c *Aggregator) Collect(ctx context.Context, rec export.Record, exp export.Batcher) {
+func (c *Aggregator) Checkpoint(ctx context.Context, desc *export.Descriptor) {
```

Comment:
missing comments, unlike the other aggregators; should discuss here that it clears the current array.